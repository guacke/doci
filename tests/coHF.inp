FCIDUMP          = co_631g_1.FCIDUMP
SOLVER           = D
MAX_ITS          = 1000
TOL              = 1.000000e-08
DAVIDSON_KEEP    = 2
DAVIDSON_MAX_VEC = 30
INITIAL          = 1