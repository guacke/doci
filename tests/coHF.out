doci -- An implementation of DOCI (Doubly Occupied Configuration Interaction)

input-file
----------

FCIDUMP          = co_631g_1.FCIDUMP

SOLVER           = DAVIDSON

MAX_ITS          = 1000

TOL              = 1.000000e-08

DAVIDSON_KEEP    = 2

DAVIDSON_MAX_VEC = 30

INITIAL          = Hartree-Fock

----------
	** executing DOCI (4 threads)
	** N = (7,7)
	** L = 28
	** basis size : 1184040
----------
--------------------------------------------------
IT	INFO	RESIDUE	ENERGY
1	0	5.498227e-01	-112.733134
2	0	1.533984e-01	-112.834220
3	0	3.076053e-02	-112.839054
4	0	5.015144e-03	-112.839215
5	0	6.715290e-04	-112.839219
6	0	7.690078e-05	-112.839219
7	0	1.025249e-05	-112.839219
8	0	1.458263e-06	-112.839219
9	0	1.960934e-07	-112.839219
10	0	2.582896e-08	-112.839219
11	0	2.986288e-09	-112.839219
Davidson converged in 11 iterations with 2.986288e-09 residue and d_energy 0.000000e+00.
elapsed time : 2.459269 sec
average time per matvec : 0.223570 sec
		 ** EXIT CODE SOLVER : 0
		 ** DOCI ENERGY : -112.8392190587
		 ** ELAPSED TIME : 2.665814
--------------------------------------------------
exiting...
